contents/teams/marketing/objectives.mdx

```diff
@@ -1,71 +1,44 @@
-- **We'll know we're successful when:**
-  - Our subscriber growth rises to over 8k per month.
-  - Everyone feels confident in shipping a newsletter issue. 
-  - We see a significant growth in Twitter and LinkedIn as sources of traffic.
-
-## Podcast
-
-- **Rationale:** As part of a marketing flywheel, a podcast stood out as an unexplored channel. And serves as a potential stepping stone to additional types of video content.
-- **Owned by:** Bijan
-- **Contributors:** James Hawkins
-- **What we're shipping:**
-  - Record 10 podcast episodes. Publish the best ones once we've settled on a format / branding etc.
-  - Test different formats (themed episodes, interviews, behind the scenes, etc.) and content
-- **We'll know we're successful when:**
-  - We learn a format and style of content that resonates
-  - We establish a sustainable cadence and guidelines for producing content
-  - People are mentioning and sharing our podcast on social
-
-## The Product Engineer Handbook (V1)
-
-- **Rationale:** We've carved out the cultural idea of the Product Engineer. We feel strongly that this is a high-leverage role for individuals to pursue and companies to support. We should provide guidance for people to become the best version of this, and earn recognition and traffic for doing so.
 - **Owned by:** Danilo
-- **Contributors:** Andy, Ian, Bijan
+- **Motivation:** We want to make it incredibly easy to integrate PostHog using Cursor and other AI coding agents. Goal is to ensure LLMs follow best practice, so users don't run into problems.
 - **What we're shipping:**
-  - To start, a roundup of all the best content we and others have written to inform the work of the product engineer.
+  - Single-prompt integrations for 3-5 popular frameworks
+  - Cursor rules integrated and updated via wizard
+  - Additional documentation for robots as experimentation recommends
 - **We'll know we're successful when:**
-  - People credit us with their career traction
-  - People link to our resources as essential reading
+  - We measure consistent usage of the wizard and related tools
+  - People are talking about the integration DX like "whoa, that's kinda neat, check it out"
 
-## Revamp 'Start here' section of docs
+## Even more getting started
 
-- **Rationale:** We need a more coherent approach to building expertise in our product. *Reasoning In PostHog* is challenging, not because the mental model doesn't make sense, but because the power is so abundant. No one has been handed quite so many tools that work this well.
 - **Owned by:** Danilo
-- **Contributors:** Andy, Ian, Bijan
+- **Motivation:** First wave of getting started guides have been a big help for the sales team especially. There's plenty more we can do, so let's keep shipping.
 - **What we're shipping:**
-  - A set of intro docs, think 5-7, that help newbies reason in PostHog and apply it to their work.
+  - Guidance to help our true believers get their companies to switch
```

Comment:
Would like to firm this up a little @daniloc. What guidance do we want to ship? An exhaustive list isn't necessary, but a guesstimate of what you're planning would be good. We can always change it later.